OpenRA.Game/Platform.cs

```diff
@@ -66,111 +74,100 @@ public static string RuntimeVersion
+					legacyUserSupportPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Personal), ".openra");
 
 					var xdgConfigHome = Environment.GetEnvironmentVariable("XDG_CONFIG_HOME");
 					if (string.IsNullOrEmpty(xdgConfigHome))
 						xdgConfigHome = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Personal), ".config");
 
-					preferredSupportDir = Path.Combine(xdgConfigHome, "openra");
+					modernUserSupportPath = Path.Combine(xdgConfigHome, "openra");
+					systemSupportPath = "/var/games/openra/";
 
 					break;
 				}
 
 				default:
 				{
-					preferredSupportDir = fallbackSupportDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Personal), ".openra");
+					modernUserSupportPath = legacyUserSupportPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.Personal), ".openra");
+					systemSupportPath = "/var/games/openra/";
 					break;
 				}
 			}
 
+			// Use a local directory in the game root if it exists (shared with the system support dir)
+			var localSupportDir = Path.Combine(GameDir, "Support");
+			if (Directory.Exists(localSupportDir))
+				userSupportPath = systemSupportPath = localSupportDir + Path.DirectorySeparatorChar;
+
 			// Use the fallback directory if it exists and the preferred one does not
-			if (!Directory.Exists(preferredSupportDir) && Directory.Exists(fallbackSupportDir))
-				return fallbackSupportDir + Path.DirectorySeparatorChar;
+			else if (!Directory.Exists(modernUserSupportPath) && Directory.Exists(legacyUserSupportPath))
+				userSupportPath = legacyUserSupportPath + Path.DirectorySeparatorChar;
+			else
+				userSupportPath = modernUserSupportPath + Path.DirectorySeparatorChar;
 
-			return preferredSupportDir + Path.DirectorySeparatorChar;
+			supportDirInitialized = true;
 		}
 
 		/// <summary>
-		/// Directory containing system-wide support files (mod metadata).
-		/// This directory is not guaranteed to exist or be writable.
-		/// Consumers are expected to check the validity of the returned value, and
-		/// fall back to the user support directory if necessary.
+		/// Specify a custom support directory that already exists on the filesystem.
+		/// Cannot be called after Platform.SupportDir / GetSupportDir have been accessed.
 		/// </summary>
-		public static string SystemSupportDir { get { return systemSupportDir.Value; } }
-		static Lazy<string> systemSupportDir = Exts.Lazy(GetSystemSupportDir);
-
-		static string GetSystemSupportDir()
+		public static void OverrideSupportDir(string path)
 		{
-			// Use a local directory in the game root if it exists (shared with the system support dir)
-			var localSupportDir = Path.Combine(GameDir, "Support");
-			if (Directory.Exists(localSupportDir))
-				return localSupportDir + Path.DirectorySeparatorChar;
+			if (supportDirInitialized)
+				throw new InvalidOperationException("Attempted to override user support directory after it has already be accessed.");
```

Comment:
Was about to merge this, but stumbled over the typo here: "(...) _been_ accessed."